app/main/views/metrics.py

```diff
@@ -32,6 +32,31 @@ def get_metrics():
     return jsonify(metrics)
 
 
+@main.route('/metrics/domains', methods=['GET'])
+def get_domain_metrics():
+    metrics = {'groups': [x.name for x in Domain.query.all()], 'columns': [],
+               'metrics': {}, 'timestamp': pendulum.now().to_iso8601_string()}
+
+    for status in ['assessed', 'unassessed']:
+        column = [status] + ([0] * len(metrics['groups']))
+        query = '''
```

Comment:
I copied one of the other alchemy metrics but the result is wrong(returns all domains with the maximum count of one domain as the identical count for each status):
```
        domain_by_status = select([Domain.name, func.count(SupplierDomain.status)])\
            .where(SupplierDomain.status == status) .group_by(Domain.name)
```

```
    "metrics": [
        {
            "assessed": 2, 
            "domain": "Content and Publishing", 
            "unassessed": 3
        }, 
        {
            "assessed": 2, 
            "domain": "Support and Operations", 
            "unassessed": 3
        }, 
...
```